### ISSUE TITLE
Does anyone have any ethical concerns/reservations with working for a commodity brokerage firm?

### ISSUE THREAD

**Issue by nelsonic** [nelsonic/proposal-draft#2]
Commodity Trading/Brokerage Firms are not know for the "_Social Benefit_" they create... They tend to be purely market (_profit_) driven. 
Some say that's a _good_ thing because their motives are very clear.

I have a _strong_ view on companies whose sole objective is to make money. 
However I think its worth _considering_ doing this project because we can:
- A) Modularize and **_Open Source**_ _much_ of the work of building a communications platform and make several modules re-useable by projects with more social aims.
- B) Give people from "A Team" **_experience**_/exposure to high-availability / low-latency applications
  (a different league from simple web apps)
- C) **_pay people well**_ for their time (_so the best people don't feel like they have to leave for other jobs..._)

For _now_ we are only preparing the proposal for the prospective client and we _expect_ he will balk at the price; we have no intention of doing this for "cheap" as The Client makes enough profit to afford to pay market rates.

If you have any concerns with us _considering_ working for a
trading company please voice them. Thanks!

For more detail please see the original brief: https://docs.google.com/document/d/1pJ-yN5lw2P-g_DqYboM8y-IpcYBANpFLpi3PZQYgiWs

**Comment by sofer** [comment]
I have no concern with dwyl _considering_ working for a commodity broker, particularly in the absence of other plausible projects. I might question dwyl _actively pursuing_ such projects.

**Comment by notenociz** [comment]
I agree with @sofer. I just want to clarify as to who will own the software? I'm not too good on the legal side but can @dwyl own the software and Company XYZ the app?

**Comment by notenociz** [comment]
Also, at the moment, are we only waiting for approval from Company XYZ before assembling a core team?

**Comment by notenociz** [comment]
Oh! We should also get [our GitHub reference](https://github.com/dwyl/github-reference/) going to help client see what's going on.

**Comment by nelsonic** [comment]
To be clear, there are _many_ "plausible projects". 
We have only _just_ been given the "_mandate_" by FAC members to handle the commercial side of things in a separate company. https://github.com/iteles/ateam/issues/22#issuecomment-149047724
We have _barely_ started and its _far_ more important to understand _exactly_ what types of organisations we _want_ to work with https://github.com/iteles/ateam/issues/28 so that we _don't_ fall into the trap of taking the _first_ project that someone approaches us to build. Which is what _all_ other "_agencies_" do and then _wonder_ why they don't have a "_lets fix the world_" culture...

We need _time_ to establish a system for securing new work. 
I am speaking to _several_ interesting organisations _informally_, but we need a _systematic_ approach to this. Our _plan_ is to build a pipeline of work but this will not happen overnight. 
Our _immediate_ concerns are:
- Deliver existing project for existing clients. (i.e. don't leave people in the lurch ... _unacceptable_)
- Develop internal systems for managing work. (Not 7 different systems... _unscalable_ )

Every single day I get offers from Financial Companies (_because of past experience_) to do work at day-rates most developers can only _dream_ about. The reason I _don't_ take any offers is because I don't feel that _any_ of the _established_ Financial companies are _solving_ a _genuine_ "**_human problem**_"...

The problem "Company XYZ" needs to solve can be defined _broadly_ as a **_Communications Problem**_. _That_ is a problem that every (_non-cave-dwelling-hermit_) Human Being has.
So `if` we can get _paid_ to solve it for one company we can apply (_a part_) of the solution to other companies. 

However, since the prospective client (_lovely guy_ BTW) approached Izaak we made the _effort_ to prepare the pitch in case other people on the Team _want_ to work on the project.

`If` Company XYZ decide that we are the best people to build their solution, we will assemble a team of people and _blow their minds_ with our execution. Then, we will get back to what _matters_.

**Comment by nelsonic** [comment]
@enociz on the subject of "ownership". The client will "own" the end result. 
_Analogy_:  they are buying a "_cake_" from us. They aren't buying the mixing bowl, spatula, baking trays or oven.

We will _use_ (_and_ **_contribute**_ _to_) **_several**_ **Open Source** components in building the solution. If unclear I can elaborate further. let me know. :+1:
